includes/Services/LabkiPackManager.php

```diff
@@ -495,11 +506,20 @@ public function installSinglePack(
 		?array $rewriteMap = null,
 		?array $manifestPages = null
 	): array {
-		$packName = $packDef['name'];
-		$version = $packDef['version'];
-		$pages = $packDef['pages'];
+		$packName = $packDef['name'] ?? '(unnamed)';
+		$version = $packDef['version'] ?? null;
```

Comment:
version is a required key in the schema https://github.com/Aharoni-Lab/labki-packs-tools/blob/4a0665e1047d469f9de6a6a98c0cd104e3590153/schema/v1_0_0/manifest.schema.json#L57